### DISCUSSION TITLE
M1 IMAC Utm Ubuntu black screen

### DISCUSSION THREAD

**Discussion by serdiogo** [utmapp/UTM#5066] [UTM Help (macOS)]
Hello,

So the problem is that I installed Ubuntu server on a utm VM, after I installed the desktop running sudo, the normal thing, after the desktop finalized I rebooted it and black screen ..........

anyone can help please?

Best!

(this is what happens before the thing blacks out...)
<img width="802" alt="Captura de ecrã 2023-02-14, às 05 07 33" src="https://user-images.githubusercontent.com/125337077/218644828-51a7b8e6-df92-4692-a772-7193dcc6d92c.png">

**Comment by ktprograms** [comment]
This could be the mesa issue: #4983 
Look at that for the suggested workaround (changing the display device)

**Reply by netojoa** [reply]
I had the same issue with Ubuntu 22.04.2 Server arm64. Your suggestion worked for me.
I changed the display device to virtio-ramfb.

**Comment by andrewmantovani** [comment]
Also if you are using LUKS/LVM you can add a serial/Built in terminal device. The screen is black because the password prompt does not show on the display. Once you enter the password on the serial connection it will boot as normal.
